tests/sentry/uptime/subscriptions/test_regions.py

```diff
@@ -34,7 +35,28 @@ class GetActiveRegionConfigsTest(TestBase):
     def test_returns_only_enabled_regions(self):
         with (
             override_settings(UPTIME_REGIONS=self.test_regions),
-            override_options({"uptime.disabled-checker-regions": ["eu"]}),
+            override_options(
+                {
+                    "uptime.checker-regions-mode-override": {
+                        "eu": UptimeSubscriptionRegion.RegionMode.INACTIVE
```

Comment:
Shouldn't this be the string representation of the enum?

Reply:
Oh does it automatically coerce when using StrEnum or something?

Reply:
It auto coerces on comparison but yeah, you're right that we should use the actual value that would be returned for the test to be accurate.

Reply:
Let's do that